scripts/llnl_scripts/llnl_lc_build_tools.py

```diff
@@ -271,6 +271,7 @@ def uberenv_create_mirror(prefix, spec, project_file, mirror_path):
     assertUberenvExists()
     cmd  = "python {0} --create-mirror -k ".format(get_uberenv_path())
     cmd += " --prefix=\"{0}\" --mirror=\"{1}\"".format(prefix, mirror_path)
+    cmd += "--spec=\"{0}\" ".format(spec)
```

Comment:
Is this missing a space? 
```suggestion
    cmd += " --spec=\"{0}\" ".format(spec)
```

Reply:
Fixed the line above it to be more consistent with all other lines.. good catch though.